wwwroot/src/pages/Landing.js

```diff
@@ -0,0 +1,260 @@
+import accessIcon from '../../public/landing-access-icon.svg';
+import laptopIcon from '../../public/landing-laptop-icon.svg';
+import triangleChild from '../img/landing/triangle-child.jpg';
+import squareChild from '../img/landing/square-child.jpg';
+import circleChild from '../img/landing/circle-child.jpg';
+
+const landing1Data = {
+  title: "Learning can't stop.",
+  copy: "Dedicated to continuing to educate our future and stay connected, no matter what."
+};
+
+const landing2Data = {
+  title: "Education wherever you are.",
+  icons: [
+    {
+      img: toolsIcon,
+      title: 'Tools for Teachers',
+      copy: 'Assign work, grade, post videos or agendas all in one platform.'
+    },
+    {
+      img: accessIcon,
+      title: 'Parent Access & Monitoring',
+      copy: "Monitor your child's school work, grades, and more."
+    },
+    {
+      img: laptopIcon,
+      title: 'Easy Learning for Students',
+      copy: "Easy to access resources and tools for learning, studying and connecting."
+    }
+  ]
+};
+
+const useStyles = makeStyles((theme) => ({
+  root: {},
+  landing1root: {
+    minHeight: '86vh',
+    height: 'fit-content',
+    paddingTop: theme.spacing(33),
+    paddingLeft: theme.spacing(1)
+  },
+  landing1CopyContainer: {
+    width: theme.spacing(55),
+    margin: 0,
+  },
+  landing1CopyTitle: {
+    fontSize: '40px',
+    lineHeight: '54px',
+    fontWeight: 600,
+    color: theme.palette.primary.main,
+  },
+  landing1CopyBody: {fontSize: '18px'},
+
+  landing1ButtonCollection: {
+    marginTop: theme.spacing(3),
+    justifyContent: 'flex-end',
+  
+    '& a': {
+      width: theme.spacing(20.25),
+      height: theme.spacing(5.25),
+      boxShadow: 'none',
+
+      '&:last-child': {marginLeft: theme.spacing(2)},
+      '& p': {
+        fontSize: '16px', 
+        fontWeight: 700,
+        letterSpacing: '0.05em'
+      }
+    }
+  },
+  landing1TriangleChild: {
+    width: '262px',
+    height: '411.28px',
+    position: 'relative',
+    top: -theme.spacing(56),
+    left: theme.spacing(53),
+    clipPath: 'polygon(50% 22%, 3% 76%, 97% 76%)'
+  },
+  landing1CircleChild: {
+    width: '233px',
+    height: '350px',
+    position: 'relative',
+    top: -theme.spacing(67),
+    left: theme.spacing(52),
+    clipPath: 'circle(116px at 50% 66%)'
+  },
+  landing1SquareChild: {
+    width: '345px',
+    height: '233.09px',
+    position: 'relative',
+    top: -theme.spacing(33.5),
+    left: theme.spacing(2.5),
+    clipPath: 'polygon(11% 0%, 11% 100%, 76% 100%, 76% 0%)'
+  },
+
+  
+  landing2root: {
+    height: '86vh',
+    paddingTop: theme.spacing(6)
+  },
+  landing2Container: {
+    textAlign: 'center',
+    [theme.breakpoints.down('lg')]: {
+      padding: '0 10%'
+    }
+  },
+  landing2Title: {
+    fontSize: '32px',
+    fontWeight: 600,
+    marginBottom: theme.spacing(9)
+  },
+  landing2ItemBox: {
+    width: theme.spacing(48),
+    margin: 0,
+
+    '& h3': {
```

Comment:
Should we move it to MuiTheme -creation on App level? Or ais this h3 styling is for this page only? Seems like this should be globally changed on the whole theme and not only in this component. Can you please move font-size, font-weight, etc into typography settings on App.js?

Reply:
I don't see the styling in usage on any other page than the Why Us page (which shares a whole section with the Landing page), and in that case I just directly import the component that this styling applies to - so this h3 styling is just for this page.

Reply:
ok, let's watch out when we work on other pages. If it requires the same styling, then we should take these stylings out to theme definition at App.js